internal/project/ata/ata_test.go

```diff
@@ -0,0 +1,310 @@
+package ata_test
+
+import (
+	"context"
+	"slices"
+	"testing"
+
+	"github.com/microsoft/typescript-go/internal/bundled"
+	"github.com/microsoft/typescript-go/internal/lsp/lsproto"
+	"github.com/microsoft/typescript-go/internal/testutil/projecttestutil"
+	"gotest.tools/v3/assert"
+)
+
+func TestATA(t *testing.T) {
+	t.Parallel()
+	if !bundled.Embedded {
+		t.Skip("bundled files are not embedded")
+	}
+
+	t.Run("local module should not be picked up", func(t *testing.T) {
+		t.Parallel()
+		files := map[string]any{
+			"/user/username/projects/project/app.js":    `const c = require('./config');`,
+			"/user/username/projects/project/config.js": `export let x = 1`,
+			"/user/username/projects/project/jsconfig.json": `{
+					"compilerOptions": { "moduleResolution": "commonjs" },
+					"typeAcquisition": { "enable": true }
+			}`,
+		}
+
+		testOptions := &projecttestutil.TypingsInstallerOptions{
+			TypesRegistry: []string{"config"},
+		}
+
+		session, utils := projecttestutil.SetupWithTypingsInstaller(files, testOptions)
+		uri := lsproto.DocumentUri("file:///user/username/projects/project/app.js")
+		content := files["/user/username/projects/project/app.js"].(string)
+
+		// Open the file
+		session.DidOpenFile(context.Background(), uri, 1, content, lsproto.LanguageKindJavaScript)
+		session.WaitForBackgroundTasks()
+		ls, err := session.GetLanguageService(context.Background(), uri)
+		assert.NilError(t, err)
+		// Verify the local config.js file is included in the program
+		program := ls.GetProgram()
+		assert.Assert(t, program != nil)
+		configFile := program.GetSourceFile("/user/username/projects/project/config.js")
+		assert.Assert(t, configFile != nil, "local config.js should be included")
+
+		// Verify that only types-registry was installed (no @types/config since it's a local module)
+		npmCalls := utils.NpmExecutor().NpmInstallCalls()
+		assert.Equal(t, len(npmCalls), 1)
+		assert.Equal(t, npmCalls[0].Args[2], "types-registry@latest")
+	})
+
+	t.Run("configured projects", func(t *testing.T) {
```

Comment:
Can you please keep the skipped tests or keep them commented or something so they are tracking pending work 

Eg deduplicate from local @types packages